vassal-app/src/main/java/VASSAL/build/module/PlayerRoster.java

```diff
@@ -622,13 +622,36 @@ protected boolean allSidesAllocated() {
 
   /**
    * @param side Name of a side to see if it's a "solo side"
-   * @return True if the side is "Solitaire", "Solo", "Moderator", or "Referee"
+   * @return True if the side is "Solo" or begins "Solitaire", "Solo:", "Moderator", "Referee" or a Side
+   * found in the Global Translatable Message Property: VassalSkipSideList (a comma-separated list).
+   * To test specifically for Referee/Moderator, use isRefereeSide()
    */
   public static boolean isSoloSide(String side) {
-    return Resources.getString("PlayerRoster.solitaire").equals(side) ||
-           Resources.getString("PlayerRoster.solo").equals(side) ||
-           Resources.getString("PlayerRoster.moderator").equals(side) ||
-           Resources.getString("PlayerRoster.referee").equals(side);
+    // FIXME: A future version of Vassal may provide a Side Type attribute in Definition of Sides that would allow this function to be rationalised.
+    // See also isRefereeSide()
+    return side.startsWith(Resources.getString("PlayerRoster.solitaire")) ||
+            side.equals(Resources.getString("PlayerRoster.solo")) ||
+            side.startsWith(Resources.getString("PlayerRoster.solo") + ":") ||
+            side.startsWith(Resources.getString("PlayerRoster.moderator")) ||
```

Comment:
Why "startsWith" here where it used to only check equals? Are we expecting "Moderator Left" and "Moderator Right"? 

If so, why are we requiring a different format here from Solo (where it requires the colon e.g. "Solo:")? And for that matter why do we only eg allow "Solo:" and not "Solitaire:"